### ISSUE TITLE
Suddenly cannot see my attachments anymore [Google Cloud Storage]

### ISSUE THREAD

**Issue by iwalucas** [jschneier/django-storages#644]
Hello, I am using the django-storage with google buckets for a time now. A few days ago I noticed it started not showing me the links to download the attachments anymore. I can upload, and see it on google storage, but not through my django application, its like I havent even uploaded it.

I am on Google App Engine - Standard env.

Any idea on why this is happening?

**Comment by peteza33** [comment]
Same issue for me. 

I can upload successfully and see it in my bucket, but the app doesn't seem to register it exists and returns 500

**Comment by sww314** [comment]
Hey guys, I am using this will Google Cloud and it is working fine for me -- I am not on Google App Engine. 
if you turn on debugging, what does stacktrace say?

**Comment by peteza33** [comment]
Tried reverting to storages 1.6.5 and its back to working like before/normal. So maybe something with the .url rework?

I'll get a stacktrace

**Comment by sww314** [comment]
If the file is not public, it will try to sign the file. I did not test the signing in GAE.

**Comment by SergeyKosarchuk** [comment]
Hello.
Is this issue still relevant? I came across same problem about a month ago. Users could upload attachments via standard FileField form, but not view them laster. I also saw new buckets created. Fixed it by migrating to newer version of django-storages (1.7.1). Seems like users was getting private links to cloud storage with out necessary permissions to view it. In newer versions have changed default approach to share file links, from permanent private to temporary public.
